versions/3.1.1.md

```diff
@@ -267,8 +267,8 @@ The following table shows how to migrate from OAS 3.0 binary data descriptions,
 
 | OAS < 3.1                            | OAS 3.1                                                                          | Comments                                                                                |
 | ------------------------------------ | -------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
-| `type: string`<br />`format: binary` | `contentMediaType: image/png`                                                    | if redundant, can be omitted, often resulting in an empty [Schema Object](#schema-object) |
-| `type: string`<br />`format: byte`   | `type: string`<br />`contentMediaType: image/png`<br />`contentEncoding: base64` | note that `base64url` can be used to avoid re-encoding the base64 string to be URL-safe |
+| `type: string`<br /><code style="white-space:nowrap">format: binary</code> | <code style="white-space:nowrap">contentMediaType: image/png</code> | if redundant, can be omitted, often resulting in an empty [Schema Object](#schema-object) |
+| `type: string`<br />`format: byte` | `type: string`<br />`contentMediaType: image/png`<br />`contentEncoding: base64` | note that `base64url` can be used to avoid re-encoding the base64 string to be URL-safe |
```

Comment:
I think it might be best to do all of the bits with `<code style...` because either someone will have a device on which the other parts wrap unexpectedly (even if I don't see how that would happen offhand) or someone will slightly tweak this and then one of the non-`<code...` bits will end up longer and wrapping.

But I'm not going to hold this up if you'd rather leave it as-is.